### ISSUE TITLE
Add option to not create a LB for 3+ masters

### ISSUE THREAD

**Issue by aleksasiriski** [vitobotta/hetzner-k3s#139]
This is a useful cost saving measure, the default should remain as is but if one would want a HA setup they don't need a Load Balancer for that, as a Load Balancer for master nodes only allows HA kubeapi (but you can just use another master's ip if the first one is down).

Every aspect of HA remains the same without the LoadBalancer since all nodes talk to each other directly and not via the LB.

**Comment by aleksasiriski** [comment]
I'll look into addinp this in a PR, I'm interested in learning Ruby / Crystal (:

**Comment by vitobotta** [comment]
Sure, looking forward to the PR. As long as we keep the current default I don't mind giving the option to avoid the lid balancer if one wants to save that cost.

**Comment by vitobotta** [comment]
@aleksasiriski Hi, if you are planning on doing a PR for this can we close this issue?

**Comment by aleksasiriski** [comment]
Sure

**Comment by aleksasiriski** [comment]
I'm a little busy currently with my studies but will start on the PR soon.

Also, just for experimenting I tried out kube-hetzner terraform and it's awful* (at least for me, maybe I just had bad luck idk):
1. Their default outbound rules break most of my containers and I needed to add extra rules that allow all traffic with no way of specifying that I don't want ANY rules added.
2. Their own installation of Traefik isn't HA and randomly breaks giving `Forbidden` when trying to access ingresses.
3. There's no way of telling the script to install additional packages and Longhorn helm install, if not using their own, doesn't work because of missing iSCSI and NFS clients, even though they claim "it's already installed".

I think they're trying to cover a lot of usecases with their options but then don't take the time to properly test them, whereas your script installs a minimal cluster and let's user's install all the needed services on their own which in turns means it's not complete from the get-go but you don't have to maintain those install scripts (or update the helm charts).
